src/Sylius/Bundle/CoreBundle/Twig/PriceExtension.php

```diff
@@ -22,44 +21,18 @@
 
 final class PriceExtension extends AbstractExtension
 {
-    public function __construct(private readonly PriceHelper|ProductVariantPricesCalculatorInterface $helper)
+    public function __construct(private readonly ProductVariantPricesCalculatorInterface $productVariantPriceCalculator)
```

Comment:
```suggestion
    public function __construct(private readonly ProductVariantPricesCalculatorInterface $productVariantPricesCalculator)
```

And what about renaming the following class:
https://github.com/Sylius/Sylius/blob/9dafe11fb2f40639fd9bb7fe2d739864c34451a6/src/Sylius/Component/Core/Calculator/ProductVariantPriceCalculator.php#L23
to match the interface name, wdyt?